gobblin-core/src/main/java/gobblin/writer/http/LimiterHttpClient.java

```diff
@@ -0,0 +1,44 @@
+package gobblin.writer.http;
+
+import java.io.IOException;
+import com.codahale.metrics.Meter;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import gobblin.http.HttpClient;
+import gobblin.util.limiter.Limiter;
+
+@Slf4j
+@AllArgsConstructor
+public class LimiterHttpClient<RQ, RP> implements HttpClient<RQ, RP> {
```

Comment:
A related question is: where should we enable throttling, `sendRequest` vs `dispatch`. `dispatch` may issue some retires, which, in my opinion, should use the same permit. 

This class is not necessary for throttling on the dispatch level.